vector/src/main/java/im/vector/app/features/settings/devices/v2/VectorSettingsDevicesFragment.kt

```diff
@@ -52,7 +67,45 @@ class VectorSettingsDevicesFragment @Inject constructor() : VectorBaseFragment<F
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
         initLearnMoreButtons()
+        initWaitingView()
+        observerViewEvents()
+    }
+
+    private fun observerViewEvents() {
+        viewModel.observeViewEvents {
+            when (it) {
+                is DevicesViewEvents.Loading -> showLoading(it.message)
+                is DevicesViewEvents.Failure -> showFailure(it.throwable)
+                is DevicesViewEvents.RequestReAuth -> Unit // TODO. Next PR
+                is DevicesViewEvents.PromptRenameDevice -> Unit // TODO. Next PR
```

Comment:
if possible it could be handy to link to tickets, although I guess the PS tickets are in jira :thinking:

Reply:
Yes, they are private atm.